From c1d9796851dbc8d835ee6833dd02376ce88011a3 Mon Sep 17 00:00:00 2001
From: Tendai Makumire <tendai.makumire@imd-tec.com>
Date: Tue, 7 Apr 2026 14:02:35 +0100
Subject: [PATCH] mxm_wifiex: add support for autoloading SDIO drivers

The SDIO device ID's were not included in
MODULE_DEVICE_TABLE() so the driver
was not being autoloaded when the
SDIO device ID was found on the SDIO
bus.
---
 mlinux/moal_sdio_mmc.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mlinux/moal_sdio_mmc.c b/mlinux/moal_sdio_mmc.c
index d216017..b07c72c 100644
--- a/mlinux/moal_sdio_mmc.c
+++ b/mlinux/moal_sdio_mmc.c
@@ -158,6 +158,8 @@ static const struct sdio_device_id wlan_ids[] = {
 	{},
 };
 
+MODULE_DEVICE_TABLE(sdio, wlan_ids);
+
 
 int woal_sdio_probe(struct sdio_func *func, const struct sdio_device_id *id);
 void woal_sdio_remove(struct sdio_func *func);
-- 
2.41.0

